Source/DafnyStandardLibraries/Makefile

```diff
@@ -2,6 +2,7 @@
 # Invoking the CLI this way just to stay platform-independent
 DAFNY = dotnet run --project ../Dafny --no-build --
 
+NO_VERIFY := false
```

Comment:
Okay I was a bit nervous about `--hidden-no-verify`, but I'm okay with it given the default state of the Makefile is still to verify. I can see how having this option for more rapid development would be super useful.